components/http/okHttp/gradle/dependencies.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
     // This dependency is used internally, and not exposed to consumers on their own compile classpath.
     implementation 'io.opentelemetry:opentelemetry-api:1.31.0'
     implementation 'io.opentelemetry:opentelemetry-context:1.31.0'
-    implementation 'io.opentelemetry:opentelemetry-semconv:1.30.1-alpha'
+    implementation 'io.opentelemetry.semconv:opentelemetry-semconv:1.22.0-alpha'
```

Comment:
can you provide the rationale here? it feels like a step back version wise

Reply:
```
Deprecated
Use io.opentelemetry.semconv.SemanticAttributes from io.opentelemetry.semconv:opentelemetry-semconv:{{version}}  instead. 
```
The previous one was the old dependency path and the second is the new one, it seems this one is fully dedicated to the semconv library. 1.22.0 is the latest for this version.

Reply:
ok thanks for the additional context here